pysqldb3/geopackage.py

```diff
@@ -343,41 +339,24 @@ def gpkg_to_shp(self,
                 count_cmd = COUNT_GPKG_LAYERS.format(full_path = os.path.join(self.path, self.gpkg_name))
                 ogr_response = subprocess.check_output(shlex.split(count_cmd.replace('\n', ' ')), stderr=subprocess.STDOUT)
                 tables_in_gpkg = re.findall(r"\\n\d+:\s(.*?)(?=\\r|\s\(.*\))", str(ogr_response)) # only allows tables names with underscores, numbers, and letters
```

Comment:
i think rather than have the bulk call this function with bulk = true, i would move the bulk logic to the gpkg_to_shp_bulk function and have this call that

Reply:
OK I have changed that in my latest update